patches/chrome-browser-permissions-permission_manager.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/permissions/permission_manager.cc b/chrome/browser/permissions/permission_manager.cc
+index af0c4c0e74fbcf78f02fe112f19724260dda456a..dc1dea1a9ec64c9d7b6344ee26be2e888f217076 100644
+--- a/chrome/browser/permissions/permission_manager.cc
++++ b/chrome/browser/permissions/permission_manager.cc
+@@ -121,6 +121,8 @@ ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
+       return CONTENT_SETTINGS_TYPE_CLIPBOARD_WRITE;
+     case PermissionType::PAYMENT_HANDLER:
+       return CONTENT_SETTINGS_TYPE_PAYMENT_HANDLER;
++    case PermissionType::AUTOPLAY:
```

Comment:
overrides is not gonna work because there are callers of it in the same file. It will end up no one calling this function.
Second idea doesn't work either unless we turn off this compile time checking
`error: enumeration value 'AUTOPLAY' not handled in switch [-Werror,-Wswitch]`
`error: 16 enumeration values not handled in switch: 'MIDI_SYSEX', 'NOTIFICATIONS', 'GEOLOCATION'... [-Werror,-Wswitch]`

Reply:
if we subclass PermissionManager and then override those caller method, it eliminates the patch but that would be too many copy logic. Not sure if it is worthy.

Reply:
@darkdh it doesn't matter if there are callers in the same file. @bbondy he can redefine `PermissionTypeToContentSetting`, the problem is actually that he can't reference the original method because it's in an anonymous namespace so he'd have to replace the entire thing

Reply:
actually didn't Chrome briefly turn off autoplay by default? Can't we just turn back on whatever they turned off? Seems odd that we would need our own permission for something that is already in chrome

Reply:
fixed in 11546353b674fb143f0ff64b6766679181297895